src/StatusIndicator/StatusIndicator.tsx

```diff
@@ -45,19 +55,17 @@ const getStatusIndicatorColours = (type, theme) => {
   return StatusIndicatorColours(theme)[type];
 };
 
-type StatusIndicatorType = "neutral" | "dark" | "danger" | "informative" | "success" | "warning" | "quiet";
```

Comment:
Is there a reason we're migrating away from union types? Enums have their use cases but they come with their own downsides: https://fettblog.eu/tidy-typescript-avoid-enums/

Reply:
We can leave the `StatusIndicatorType`. Just can transform it to:
```
type StatusIndicatorType = keyof typeof StatusIndicatorType;
```

Reply:
I added and exported enum, because we want to use values directly from NDS in the ECO.